athena-dynamodb/README.md

```diff
@@ -10,15 +10,17 @@ This connector enables Amazon Athena to communicate with DynamoDB, making your t
 
 The Athena DynamoDB Connector exposes several configuration options via Lambda environment variables. More detail on the available parameters can be found below.
 
-1. **spill_bucket** - When the data returned by your Lambda function exceeds Lambda’s limits, this is the bucket that the data will be written to for Athena to read the excess from. (e.g. my_bucket)
-2. **spill_prefix** - (Optional) Defaults to sub-folder in your bucket called 'athena-federation-spill'. Used in conjunction with spill_bucket, this is the path within the above bucket that large
+1. **SpillBucket** - When the data returned by your Lambda function exceeds Lambda’s limits, this is the bucket that the data will be written to for Athena to read the excess from. (e.g. my_bucket)
```

Comment:
None of these had to be renamed, and doing so will make the connector not pick these environmental variables up. Ex. As an environment variable its supposed to be `spill_prefix` but in the yaml template its named `SpillPrefix`. And these config params are environmental variables the user can set.